### PR TITLE
PackageTracking: Added remaining regexes

### PR DESCRIPTION
**Pull request by Owlree**
## Description of new Instant Answer, or changes

Adds the regexes from the remaining tracking goodies to the package tracking spice.

## Related Issues and Discussions

Fixes https://github.com/duckduckgo/zeroclickinfo-goodies/issues/3973

## People to notify

@moollaza 

---

Instant Answer Page: https://duck.co/ia/view/package_tracking

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Spice/PackageTracking.pm [hunk 1]
```diff
@@ -74,6 +74,43 @@ triggers query_nowhitespace => qr/^
...
+                                (?:
+                                    [\d]{9,} |
+                                    [A-Z]{2}\d{9}CA)
+                                )
+                                $/xi;
+
+## DHL
+triggers query_nowhitespace_nodash => qr/^
+                                (?:
+                                    \d{9,} |
```
Comment (moollaza) [inline lib/DDG/Spice/PackageTracking.pm]: Is there no upper limit on DHL codes?

Comment (Owlree) [inline lib/DDG/Spice/PackageTracking.pm]: I think is should be, but I cannot find a reference on their website. I can only find this:

> The DHL Express Waybill for Time Definite shipments is a 10 digit numeric number 

But I do not now about other types of tracking numbers.

Comment (moollaza) [inline lib/DDG/Spice/PackageTracking.pm]: @Owlree okay, let's go with 10 digits for now. If we get complaints about the IA not triggering, we can update accordingly.

File: lib/DDG/Spice/PackageTracking.pm [hunk 2]
```diff
@@ -74,6 +74,43 @@ triggers query_nowhitespace => qr/^
...
+                                    \d{10} |
```
Comment (moollaza) [inline lib/DDG/Spice/PackageTracking.pm]: Looking online, I found a couple other formats that we should support:

- https://www.trackingex.com/dhl-global-mail-asia-tracking.html
- https://www.trackingex.com/dhlbenelux-tracking.html

**Comment by moollaza** [comment]
@Owlree thanks, I've updated this a bit with tests, and tighter triggering for CandaPost 👍

**Comment by bsstoner** [comment]
This was released to production 🚀

**Pull request merged**